README.md

```diff
@@ -10,16 +14,36 @@
 [![python version](https://img.shields.io/badge/python-3.8%20%7C%203.9%20%7C%203.10%20%7C%203.11-blue)](https://img.shields.io/badge/python-3.8%20%7C%203.9%20%7C%203.10%20%7C%203.11-blue)
 [![scikit-learn supported versions](https://img.shields.io/badge/sklearn-1.0%20%7C%201.1%20%7C%201.2%20%7C%201.3-blue)](https://img.shields.io/badge/sklearn-01.0%20%7C%201.1%20%7C%201.2%20%7C%201.3-blue)
 
-With Intel(R) Extension for Scikit-learn you can accelerate your Scikit-learn applications and still have full conformance with all Scikit-Learn APIs and algorithms. This is a **free software AI accelerator** that brings over **10-100X** acceleration across a variety of applications. And you do not even need to change the existing code!
+---
+</h3>
+
+<div align="left">
+
+## Overview
+
+Intel(R) Extension for Scikit-learn is a **free software AI accelerator** designed to deliver over **10-100X** acceleration to your existing scikit-learn code.
+The software acceleration is achieved with vector instructions, AI hardware-specific memory optimizations, threading, and optimizations for all upcoming Intel(R) platforms at launch time.
 
-## How it works?
 
-Intel(R) Extension for Scikit-learn offers you a way to accelerate existing scikit-learn code.
-The acceleration is achieved through **patching**: replacing the stock scikit-learn algorithms with their optimized versions provided by the extension.
+With Intel(R) Extension for Scikit-learn, you can:
 
-One of the ways to patch scikit-learn is by modifying the code. First, you import an additional Python package (`sklearnex`) and enable optimizations via `sklearnex.patch_sklearn()`. Then import scikit-learn estimators:
+* Speed up training and inference by up to 100x with the equivalent mathematical accuracy
+* Benefit from performance improvements across different Intel(R) hardware configurations
+* Integrate the extension into your existing Scikit-learn applications without code modifications
+* Continue to use the open-source scikit-learn API
+* Enable and disable the extension with a couple of lines of code or at the command line
+
+Intel(R) Extension for Scikit-learn is also a part of [Intel(R) AI Tools](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-analytics-toolkit.html). 
+
+## Acceleration
+
+![](https://raw.githubusercontent.com/intel/scikit-learn-intelex/master/doc/sources/_static/scikit-learn-acceleration-2021.2.3.PNG)
```

Comment:
Might be worth renaming file?  because it's not 2021.2.3 version in new measurements.

Also can we add relative path here?

Reply:
Renamed. The relative path can me used only if the image is in the same folder as readme file

Reply:
ok, then we should keep previous version and not delete it, as it have links to it from multiple places including pypi package page